assets/demo-todo-app.spec.ts

```diff
@@ -207,7 +207,7 @@ test.describe('Editing', () => {
     const todoItem = page.getByTestId('todo-item').nth(1);
     await todoItem.dblclick();
     await expect(todoItem.getByRole('checkbox')).not.toBeVisible();
-    await expect(todoItem.locator('label')).not.toBeVisible();
+    await expect(todoItem.getByText(TODO_ITEMS[1])).not.toBeVisible();
```

Comment:
```suggestion
    await expect(todoItem.locator('label', {
      hasText: TODO_ITEMS[1],
    })).not.toBeVisible();
```

seems more clearer here.